specification/evaluation-context/evaluation-context.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "id": "Requirement 3.1",
+        "clean id": "requirement_3_1",
+        "content": "The `evaluation context` structure MUST define a required `targeting key` field of type string, identifying the subject of the flag evaluation.",
```

Comment:
I think this is in conflict with Hooks 1.1 which says: 
> Hook context MUST provide: the flag key, flag type, evaluation context, and the default value.

If you have to provide an `evaluation context` and an `evaluation context MUST define a required 'targeting key' field`, then that conflicts with Flag Evaluation 1.7 which says context is optional.

> The client MUST provide methods for flag evaluation, with parameters flag key (string, required), default value (boolean | number | string | structure, required), evaluation context (optional), and evaluation options (optional), which returns the flag value.

Previous to this requirement I was specifying an empty EvaluationContext if one didn't exist. Can/should I just generate a random value as the targeting key? Something else?

Reply:
Seems like another duplicate suggestion, see my other response.